src/Support/Mail/MailSenderService.php

```diff
@@ -36,6 +36,11 @@ public function setMailer(Swift_Mailer $mailer): void
      */
     public function send(string $subject, string $text, $recipients, $replyTo = null): int
     {
+        // do not try to send mail for local test instance
+        $url = $this->config->get('loginurl');
```

Comment:
`$this->config->get('loginurl')` ist auch auf `test` nicht gesetzt und dort ist es hilfreich, wenn wir Emails versenden zum testen